ui/src/components/Common/Demo/EditNode.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <p>
-    <Button :label="buttonLabel" primeClass="rounded" @click="onBackClick"></Button>
+    <feather-back-button @click="onBackClick">Back To Node Table</feather-back-button>
```

Comment:
Let's keep the same casing for all components in the template <FeatherBackButton>

Reply:
Done.